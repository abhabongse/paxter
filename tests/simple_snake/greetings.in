@load!{string}\
@!##{
def add_one(num):
    return num + 1
}##\
\
@strip{   Hello, my full name is @tag[label="b"]{John Smith}   }.
I am currently @age years old, and by this time next year I will be @{age + 1} years old.
My email is @"john@example.com".
@!{age_next_year = add_one(age)}\

Do you know that 1 + 1 = @{1 + 1}? @strip[]{ Yep }.

@!##{
import itertools

counter = itertools.count(start=1)
}##\
\
Let's count: @{next(counter)}, @{next(counter)}, @{next(counter)}.
@rjust[width=12,fillchar="_"]#<{<{}>}>#
