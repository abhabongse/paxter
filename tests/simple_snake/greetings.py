name = "John Smith"
age = 47
occupation = "Student"


def tag(text: str, label: str) -> str:
    return f"<{label}>{text}</{label}>"
