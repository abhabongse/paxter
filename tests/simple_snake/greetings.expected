Hello, my full name is <b>John Smith</b>.
I am currently 47 years old, and by this time next year I will be 48 years old.
My email is john@example.com.

Do you know that 1 + 1 = 2? Yep.

Let's count: 1, 2, 3.
________<{}>
